docs/tutorials/five-minutes-to-feature-flags.md

```diff
@@ -0,0 +1,188 @@
+---
+sidebar_position: 1
+id: five-minutes-to-feature-flags
+title: Five Minutes to Feature Flags
+---
+We’re going to add feature flagging to a Node service in under five minutes using OpenFeature, the open, vendor-agnostic feature flagging SDK.
+
+We’ll be working with a simple [express][express] server, but if you're familiar with JavaScript and Node you should be able to follow along.
+
+## Setup
+
+Before we jump into the tutorial, let's quickly get everything setup. You'll need [Git](https://git-scm.com/) and [NodeJS](https://nodejs.org/) version 16 or newer. After that, run the following commands from your favorite terminal.
+
+```bash
+git clone https://github.com/open-feature/five-minutes-to-feature-flags.git && \
+  cd five-minutes-to-feature-flags && \
+  npm install
+```
+
+## Hello, world
+
+Here’s the service we’ll be working on:
+
+```js reference title="01_vanilla.js"
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/01_vanilla.js
+```
+
+Pretty much the most basic express server you can imagine - a single endpoint at `/` that returns a plaintext “Hello, world!” response.
+
+Let's start the service by running `node 01_vanilla.js`. Now you can test to make sure it works:
+
+```bash
+$> curl http://localhost:3333
+Hello, world!
+```
+
+Yep, looks good! Go ahead and stop the server using `Ctrl` + `C`.
+
+## With cows, please
+
+Let’s imagine that we’re adding a new, experimental feature to this hello world service. We’re going to upgrade the format of the server’s response, using [cowsay][cowsay]!
+
+However, we’re not 100% sure that this cowsay formatting is going to work out, so for now we’ll protect it behind a conditional:
+
+```js reference title="02_basic_flags.js"
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/02_basic_flags.js
+```
+
+Now let's start the server with our basic flag configuration by running `node 02_basic_flags.js`. By default, the service continues to work exactly as it did before, but if we change `withCow` to `true` then the response comes in an exciting new format:
+
+:::tip
```

Comment:
I got confused by having this tip immediately follow the sentence saying that the response would be in a new format - I thought for a moment that the tip was what the output should look like. 

Could we put the tip _after_ the sample output?